lib/mutant/ast/meta/send.rb

```diff
@@ -21,6 +21,17 @@ class Send
 
         public :arguments
 
+        # Test if node is defining a proc
+        #
+        # @return [Boolean]
+        def proc?
+          !receiver && selector.equal?(:proc) ||
```

Comment:
I think you should split this up into 2 private methods: `naked_proc?` and `proc_new?` or something.